Banking/src/main/java/com/example/Banking/model/User.java

```diff
@@ -0,0 +1,31 @@
+package com.example.Banking.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "Users")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class User {
```

Comment:
I recommend it to be consistent with the table name either Users or User for both table and class name.